mod/metrics/set/type/metric/answers.rb

```diff
@@ -70,16 +71,23 @@ def value_type_code
   value_type_card.item_cards.first&.codename || default_value_type_code
 end
 
+def value_cardtype_code
+  :"#{value_type_code}_value"
+end
```

Comment:
can we document somewhere what all these variants of compositions of "value", "code", and "type" mean and what kind of value they hold?